tests/conftest.py

```diff
@@ -249,61 +249,67 @@ def virtualenv(
+        env_bin_dir = "bin"
+        if sys.platform == "win32":
+            env_bin_dir = "Scripts"
 
-    # Create temporary venv
-    tmp_path = tmp_path_factory.mktemp(test_name) / "venv"
-    # TODO[HIGH]: Implement for use outside of a base virtual environment
-    #  If this happens outside of a virtual environment the complete
-    #  /usr space is cloned
-    clone_virtualenv(
-        src_dir=str(orig_sys_executable.parent.parent), dst_dir=str(tmp_path)
-    )
+        # Activate venv
+        activate_this_file = tmp_path / env_bin_dir / "activate_this.py"
 
-    env_bin_dir = "bin"
-    if sys.platform == "win32":
-        env_bin_dir = "Scripts"
+        if not activate_this_file.is_file():
+            raise FileNotFoundError(
+                "Integration tests don't work for some local "
+                "virtual environments. Use virtualenv for "
+                "your virtual environment to run integration "
+                "tests"
+            )
 
-    # Activate venv
-    activate_this_file = tmp_path / env_bin_dir / "activate_this.py"
+        execfile(str(activate_this_file), dict(__file__=str(activate_this_file)))
 
-    if not activate_this_file.is_file():
-        raise FileNotFoundError(
-            "Integration tests don't work for some local "
-            "virtual environments. Use virtualenv for "
-            "your virtual environment to run integration "
-            "tests"
-        )
+        # Set new system executable
+        sys.executable = tmp_path / env_bin_dir / "python"
 
-    execfile(str(activate_this_file), dict(__file__=str(activate_this_file)))
+        yield tmp_path
+        # Reset system executable
+        sys.executable = orig_sys_executable
 
-    # Set new system executable
-    sys.executable = tmp_path / env_bin_dir / "python"
+        # Switch back to original venv
+        activate_this_f = Path(orig_sys_executable).parent / "activate_this.py"
 
-    yield tmp_path
-    # Reset system executable
-    sys.executable = orig_sys_executable
+        if not activate_this_f.is_file():
+            raise FileNotFoundError(
+                "Integration tests don't work for some local "
+                "virtual environments. Use virtualenv for "
+                "your virtual environment to run integration "
+                "tests"
+            )
+        execfile(str(activate_this_f), dict(__file__=str(activate_this_f)))
 
-    # Switch back to original venv
-    activate_this_f = Path(orig_sys_executable).parent / "activate_this.py"
-
-    if not activate_this_f.is_file():
-        raise FileNotFoundError(
-            "Integration tests don't work for some local "
-            "virtual environments. Use virtualenv for "
-            "your virtual environment to run integration "
-            "tests"
-        )
-    execfile(str(activate_this_f), dict(__file__=str(activate_this_f)))
+    else:
+        yield ''
 
 
 def pytest_addoption(parser):
```

Comment:
```suggestion
def pytest_add_option(parser):
```

Should this be like this?

Reply:
This function is recommended like this in the pytest [docs](https://docs.pytest.org/en/6.2.x/example/simple.html) - for ease of use and for finding resources I would leave it like this .